src/modules/app/routes/disclaimer.js

```diff
@@ -3,11 +3,11 @@
 import disclaimerEndpoint from '../../endpoint/endpoints/disclaimer'
 import { createAction } from 'redux-actions'
 
-import type { Action, Dispatch, GetState } from 'redux-first-router/dist/flow-types'
+import type { Dispatch, GetState } from 'redux-first-router'
 
 export const DISCLAIMER_ROUTE = 'DISCLAIMER'
 
-export const goToDisclaimer = (city: string, language: string): Action => createAction(DISCLAIMER_ROUTE)({city, language})
+export const goToDisclaimer = (city: string, language: string) => createAction(DISCLAIMER_ROUTE)({city, language})
```

Comment:
Why did you remove the Action return type?

Reply:
Not sure what the type here is. Not needed by the linting rules.